demos/custom-tabs-headers/src/main/java/com/google/androidbrowserhelper/demos/customtabsheaders/MainActivity.java

```diff
@@ -74,10 +81,18 @@ public void onServiceDisconnected(ComponentName componentName) {
             }
         };
 
-        String packageName = CustomTabsClient.getPackageName(MainActivity.this, null);
+        //Add package names for other browsers that support Custom Tabs and custom headers.
+        List<String> packageNames = Arrays.asList(
```

Comment:
Could you please:

1. Add `com.google.android.apps.chrome` (local builds of Chrome) to the list.
2. Reversing the list order, so we go from least stable to most. (Generally if someone has a local build of Chrome installed, they want their apps to use that.)